src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -306,10 +308,10 @@ Putting It All Together
    build a picture of the current weather. Our dummy weather forecast
    will get wind observations from four weather stations:
 
-   * Aldergrove
-   * Camborne
-   * Heathrow
-   * Shetland
+   * Aldergrove (Near Belfast in NW of the UK)
```

Comment:
I think the context is handy for non UK people.

Reply:
Hah I know that one, I thought Camborne was in Australia though :)

Reply:
Hah a quick google and I was mistaken with Canberra

Reply:
I'm pretty sure there will be one or more Cambornes in Australia, either in Tasmania or Queensland, since that's where the Cornish went.